jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildConfiguration.java

```diff
@@ -62,6 +63,19 @@ public Builder setTargetImage(@Nullable ImageReference imageReference) {
       return this;
     }
 
+    public Builder setBaseAndTargetImage(String baseImage, String targetImage)
```

Comment:
I think generally in a builder pattern, you'd want to have a setter for each individual field.